stl/inc/numeric

```diff
@@ -859,7 +859,11 @@ _NODISCARD constexpr auto _Abs_u(const _Arithmetic _Val) noexcept {
 
 // FUNCTION TEMPLATE _Stl_bitscan_forward
 template <class _Unsigned>
-constexpr unsigned long _Stl_bitscan_forward(_Unsigned _Mask) noexcept {
+_NODISCARD constexpr unsigned long _Stl_bitscan_forward(_Unsigned _Mask) noexcept {
+#ifdef __cpp_lib_bitops // TRANSITION, VSO-1020212
+    return static_cast<unsigned long>(_Countl_zero(static_cast<_Unsigned>(~_Mask)));
+

```

Comment:
I believe this empty line is unnecessary:

```suggestion
```

Reply:
I was soo close